torch/_dynamo/variables/user_defined.py

```diff
@@ -77,8 +78,11 @@ def call_function(
 
         options = VariableTracker.propagate(self, args, kwargs.values())
 
-        if self.value is torch.autograd.profiler.profile:
-            return ProfilerContextWrapperVariable()
+        if self.value in (
+            contextlib.nullcontext,
+            torch.autograd.profiler.profile,
```

Comment:
do we still need this line?

Reply:
Yes, it will be used to rewrite ```with torch.autograd.profiler.profile()``` in users' code.